corehq/messaging/smsbackends/infobip/views.py

```diff
@@ -0,0 +1,32 @@
+import json
+from corehq.apps.sms.api import incoming as incoming_sms
+from corehq.apps.sms.views import IncomingBackendView
+from corehq.messaging.smsbackends.infobip.models import SQLInfobipBackend
+from django.http import HttpResponse
+
+
+class InfobipIncomingMessageView(IncomingBackendView):
+    urlname = 'infobip_message'
+
+    @property
+    def backend_class(self):
+        return SQLInfobipBackend
+
+    def post(self, request, api_key, *args, **kwargs):
+        request_body = json.loads(request.body)
+        for message in request_body.get('results'):
+            message_sid = message.get('messageId')
+            from_ = message.get('from')
+            message_content = message.get('message')
+            if message_content.get('type') == 'TEXT':
```

Comment:
does infobip support other message types? might be nice to extract the caption from images, etc.

Reply:
Currently we only have sandbox environment enabled on infobip which is limiting us in developing/testing. but I'll definitely return to this section in a few days as soon as our full access to infobip gets approved. I am adding TODO message. I am also leaving a comment on the PR about future plans.